.github/workflows/release.yaml

```diff
@@ -1,13 +1,58 @@
-name: publish
+name: Test & Publish PyPi release
 
 on:
   release:
     types: [created]
+  workflow_dispatch:
+    inputs:
+      qhub_tag:
+        description: 'Tag for test PyPI release'
+        required: True
```

Comment:
```suggestion
```
We were using `workflow_dispatch` in the past so that we could manually trigger the test-release workflow, since we have them both in the same workflow, I don't think we need this any more.